src/comments.ts

```diff
@@ -17,17 +17,17 @@ export const HadError = (user: string | undefined, error: string) => ({
 });
 
 export const CIFailed = (slug: string, user: string, ciUrl: string) => ({
-    tag: `gh-actions-complaint-${slug}`,
+    tag: `gh-actions-complaint-${slug.slice(0, 7)}`,
```

Comment:
Would be better to send this the trimmed version so this code doesn't need to change.

Reply:
Done :heavy_check_mark: